src/terra-core/TerraCoreDataModel.ttl

```diff
@@ -968,6 +977,16 @@ TerraCore:hasAlpha2Code
   rdfs:label "hasAlpha2Code" ;
   rdfs:range xsd:string ;
 .
+TerraCore:hasAnatomicalRegion
+  a owl:DatatypeProperty ;
+  rdfs:comment "Many groups are convening to define Common Coordinate Frameworks for specific organs/species.  This will be a rapidly evolving area in the near future.  Thus allowing text or URI as we wait for the field to mature allows us to support what we hope will at a minimum be controlled vocabulary under discussion of these groups." ;
+  rdfs:domain TerraCore:BioSample ;
+  rdfs:label "hasAnatomicalRegion" ;
+  rdfs:range xsd:anyURI ;
```

Comment:
Out of curiosity, what is the difference between this `xsd:anyURI` range and the `dct:URI` range that's used on hasDevelopmentalStage?

Reply:
They should map to one another.  xsd is from the namespace for XML Schema and dct is the Dublin Core Terms which is a foundational ontology.  Both are likely used by anyone who might use a data model and both are required by this data model.  On the other hand, we should be consistent.  Looks like I've used it inconsistently.  Would you write up a ticket to define the strategy and update all our data models.  We should not modify others.  Good catch!

Reply:
Sounds good! I'll write up a ticket. And thanks for the explanation!

Reply:
Here's the ticket I made: https://broadinstitute.atlassian.net/browse/DSPDC-1147

And feel free to add anything I left out.